spec/support/contracts/json_schema_test/app/publish.mson

```diff
@@ -28,4 +28,7 @@ A post that only has a title, but no text body. Weird, right?
 - tags: (array[App::Tag]) - An array with a custom type
 - multiple_props: (array[PropA, PropB, string]) - A multitype array
 - comments: (array[string]) - An array with a primitive type
+- assignee: (enum[string])
```

Comment:
I started adding the enums and then thought it'd be easier if I refactor this a bit. This was a leftover 🙄